test/prism/ruby/parser_test.rb

```diff
@@ -179,6 +179,25 @@ def test_it_block_parameter_syntax
       assert_equal(it_block_parameter_sexp, actual_ast.to_sexp)
     end
 
+    def test_it_assignment_syntax
```

Comment:
This is for the parser gem translation and is not necessary to add as a test. The fixture you added should generate a snapshot instead that you should commit to test the prism behaviour.

Reply:
Thank you for pointing this out.
It appears I misunderstood and added the test incorrectly. I've removed this test case.

Reply:
Thank you for the comment.
I forgot to add the snapshot, and I've pushed it now.